docs/src/main/asciidoc/dev-ui.adoc

```diff
@@ -15,37 +15,45 @@ include::_attributes.adoc[]
 This guide covers the Dev UI v2, which is the default from Quarkus 3 onwards.
 ====
 
-This guide covers the Quarkus Dev UI for xref:building-my-first-extension.adoc[extension authors].
-
-Quarkus ships with a Developer UI, which is available in dev mode (when you start
-quarkus with `mvn quarkus:dev`) at http://localhost:8080/q/dev-ui[/q/dev-ui] by default. It will show you something like
-this:
+Quarkus ships with a Developer UI which is available in dev mode and provides a set of tools to help you develop your application.
 
-image::dev-ui-overview-v2.png[alt=Dev UI overview,role="center"]
+When you start quarkus with `mvn quarkus:dev` the Dev ui is available at http://localhost:8080/q/dev-ui[/q/dev-ui] by default.
 
 It allows you to:
 
 - quickly visualize all the extensions currently loaded
-- view extension statuses and go directly to extension documentation 
+- view extension statuses and go directly to extension documentation
 - view and change `Configuration`
 - manage and visualize `Continuous Testing`
 - view `Dev Services` information
 - view the Build information
 - view and stream various logs
 
+
+_In de prod QUARKUS_PROFILE the Dev ui is not available, see the documentation on how https://quarkus.io/guides/config-reference#default-profiles[configure it]_
```

Comment:
I'm not sure what you're trying to say here? We already state previously that the Dev UI is available in dev mode.

Also this guide is for extension authors, not for general considerations about the Dev UI.